entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md

```diff
@@ -3489,7 +3489,7 @@ The log level can be changed back to `Information` if desired:
 
 ### `IEntityEntryGraphIterator` is publicly usable
 
-In EF,  the <xref:Microsoft.EntityFrameworkCore.ChangeTracking.IEntityEntryGraphIterator> service can be used by applications. This is the service used internally when discovering a graph of entities to track, and also by <xref:Microsoft.EntityFrameworkCore.ChangeTracking.ChangeTracker.TrackGraph%2A>. Here's an example that iterates over all entities reachable from some starting entity:
+In EF7, the <xref:Microsoft.EntityFrameworkCore.ChangeTracking.IEntityEntryGraphIterator> service can be used by applications. This is the service used internally when discovering a graph of entities to track, and also by <xref:Microsoft.EntityFrameworkCore.ChangeTracking.ChangeTracker.TrackGraph%2A>. Here's an example that iterates over all entities reachable from some starting entity:
```

Comment:
Did we discuss "EF7" vs "EF 7"?

Reply:
I don't think so. EF7 comes after EF6. 🦆